crates/bevy_sprite/src/texture_atlas.rs

```diff
@@ -22,7 +22,7 @@ pub struct TextureAtlas {
 }
 
 #[derive(Component, Debug, Clone, TypeUuid, Reflect)]
-#[uuid = "7233c597-ccfa-411f-bd59-9af349432ada"]
+#[uuid = "0db7e8a5-658d-4b3f-8194-670c4562c0d1"]
```

Comment:
actually `TextureAtlasSprite` doesn't need to derive `TypeUuid`, it's not an asset and it should be removed

Reply:
`Sprite` also doesn't need it, and it was the other duplicate uuid so `TextureAtlas` can keep his.